### PR TITLE
SSP DDs 1-3

### PR DESCRIPTION
**Pull request by bmaclach**
This PR updates the first 3 DDs in SSP. These are the ones that had multiple equations in the manual version, so I have split them up into a DD per equation as per discussion in #1174.

### CODE REVIEW

File: code/stable/ssp/SRS/SSP_SRS.tex [hunk 1]
```diff
@@ -1435,9 +1435,159 @@ \subsubsection{Data Definitions}
...
+0, & {y_{wt,i}}\leq{}{y_{slip,i}}
+\end{cases}
+                                    \end{displaymath}
+                                    \\ \midrule \\
+                                    Description & \begin{symbDescription}
+                                                  \item{${{U^{R}}_{b}}$ is the right base hydrostatic force on a slice ($\frac{\text{N}}{\text{m}}$)}
+                                                  \item{${ℓ_{b}}$ is the total base length of a slice (m)}
+                                                  \item{$i$ is the index (Unitless)}
+                                                  \item{${y_{wt}}$ is the y ordinate (m)}
+                                                  \item{${y_{slip}}$ is the y ordinate (m)}
```
Comment (smiths) [inline code/stable/ssp/SRS/SSP_SRS.tex]: I am going to create a separate issue about the use of the term y ordinate.  I think we should just say y coordinate.  y ordinate is not a term that is used, as far as I can tell.  ordinate is used as a name of the vertical axis, but y ordinate is redundant.  I do not want to confuse this pull request with this issue, so I'll make the issue a separate one.

**Comment by JacquesCarette** [comment]
Can you merge master in? There are conflicts now.

**Comment by bmaclach** [comment]
Merged master, this should be good to go.

**Comment by JacquesCarette** [comment]
As soon as Travis agrees with you...

**Pull request merged**